www-src/app/collections/files.coffee

```diff
@@ -18,21 +18,20 @@ module.exports = class FileAndFolderCollection extends Backbone.Collection
         @query = options.query
         @notloaded = true
 
+
     isSearch: -> @path is undefined
 
-    # search use temporary view
+
     search: (callback) ->
-        params =
-            query: @query
-            fields: ['name']
-            include_docs: true
+        app.replicator.db.query 'FilesAndFolder', (err, all) =>
+            results = all.rows.filter (row) =>
+                return row.key[1].indexOf(@query) isnt -1
```

Comment:
true. 
The advice of [Cozy-guideline](https://github.com/cozy/cozy-guidelines) about this is fun : ` the return keyword should be explicit when needed `